README.md

```diff
@@ -14,14 +14,17 @@ Suggested evaluation: `+` means yes, `-` means no, `n/a` means does not apply, e
 | --------------------------------------------------------------------------- | ------------- | ----------- | ------- | ---- | ------ | ---- | --- | ------ | -- | ------------- |
 | **Interfaces**                                                              |               |             |         |      |        |      |     |        |    |               |
 | Runs in browser                                                             |               |             |+        |      |        |      |     |        |    |               |
-| Runs in notebook                                                            |               |             |+        |      |        |      |     |        |    |               |
-| Exposes API                                                                 |               |             |+        |      |        |      |     |        |    |               |
+| Runs in notebook (e.g., via JupyterLab extension)                           |               |             |+        |      |        |      |     |        |    |               |
+| Exposes API                                                                 |               |             |+ (Python and HTTPS) |      |        |      |     |        |    |               |
 | Scriptable (e.g. functions/buttons/menus)                                   |               |             |+        |      |        |      |     |        |    |               |
 | **Image formats**                                                           |               |             |         |      |        |      |     |        |    |               |
 | HiPS maps                                                                   |               |             |+        |      |        |      |     |        |    |               |
 | FITS files                                                                  |               |             |+        |      |        |      |     |        |    |               |
 | PNG/JPG images                                                              |               |             |+        |      |        |      |     |        |    |               |
 | **Image layer features**                                                    |               |             |         |      |        |      |     |        |    |               |
+| Change quantitative stretch                                                 |               |             |+        |      |        |      |     |        |    |               |
+| Change color mapping                                                        |               |             |+        |      |        |      |     |        |    |               |
+| Construct three-color displays from combinations of images                  |               |             |+        |      |        |      |     |        |    |               |
```

Comment:
They don't distinguish, as we do, between catalog data points on an image and what we call "markers"; I think that might be confusing to try to add to the table because the group might not understand that there's a distinction.

I'm adding a reference to regions to the "footprints" row, for now, and I've added the distance tool.